README.md

```diff
@@ -126,6 +126,14 @@ Modules a, b, and c would all execute in order up until the first await in each
 
 Many of the use cases for top-level `await` are for bootstrapping an application. Enforcing that top-level `await` could only be used inside of a module without exports would allow individuals to use many of the patterns they would like to use in application bootstrapping while avoiding the edge cases of graph blocking or deadlocks.
 
+Note that variant C is compatible with both A and B above, though A and B are mutually exclusive. In other words, we may consider variants A, B, AC, or BC, but not (for example) AB or ABC.
```

Comment:
So this turns out to potentially not be true. With the current text on variant C modules with no exports that are imported solely for side effects could still be imported anywhere in the tree. As such we would need to make a decision regarding how to handle the case of multiple siblings. For example

Module A
```
import './b'
import './c'
import './d'
```

Module B -> D
```
import fetch from 'fetch'
global.[b-d] = await fetch('some-url');
```

Reply:
OK, I can take this part out.